.pre-commit-config.yaml

```diff
@@ -6,3 +6,11 @@
     - id: validate-toml
     - id: no-go-testing
     - id: no-go-testing
+- repo: local
+  hooks:
+    - id: copyright_checker
+      name: copyright_checker
+      entry: python ./scripts/copyright.py
+      language: system
+      files: \.(go|py|sh)$
```

Comment:
Does `sh` includes `*.sh` files in `PROJECT_ROOT/scripts`?

Reply:
The script currently do not add copyright header if the first line starts with `#!`. Or should we add copyright string after `#!/bin/bash`?

Reply:
Let's add it after `#!/bin/bash`. Like [`go/src/run.bash`](https://github.com/golang/go/blob/master/src/run.bash)